pubspec.lock

```diff
@@ -163,6 +163,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.16.1"
+  js:
```

Comment:
please revert the changes made in pubspe.lock
Also pull the latest code and remove the conflicts